### PR TITLE
Use Ramp for embedded player

### PR DESCRIPTION
**Pull request by masaball**
Related issue: #5732

### CODE REVIEW

File: app/views/layouts/embed.html.erb [hunk 1]
```diff
@@ -28,6 +24,8 @@ Unless required by applicable law or agreed to in writing, software distributed
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <%= favicon_link_tag %>
     <%= stylesheet_link_tag "application", media: "all" %>
+    <%= stylesheet_pack_tag 'application' %>
+    <%= javascript_pack_tag 'application' %>
```
Comment (cjcolvar) [inline app/views/layouts/embed.html.erb:28]: I'm slightly concerned that including the whole application JS/CSS packs will make for a large download for only the embed.  I wonder if we can create a new pack that is just what is needed for the embed player.  Or maybe the size isn't that large and this isn't really a problem.

**Review by cjcolvar — COMMENTED** [review]
Nice!  Fewer lines than the original!

**Review by Dananji — DISMISSED** [review]
Looks great! 👍

File: app/views/layouts/embed.html.erb (hunk 1 above)
Comment (masaball) [reply app/views/layouts/embed.html.erb:28]: It looks like our compiled assets come out to around 8MB, which feels like a lot of extra data for just a player? I'll take a look at creating a new pack and trimming that down.

Comment (cjcolvar) [reply app/views/layouts/embed.html.erb:28]: Thanks!  Good luck! :crossed_fingers:

**Pull request merged**